### PR TITLE
Comment out dangerous commands, and move terminus alias

### PR DESCRIPTION
**Pull request by amitaibu**
## The Problem/Issue/Bug:

Comment out dangerous commands in Pantheon integration.

## Related Issue Link(s):

https://github.com/drud/ddev/pull/2957#issuecomment-826106788

### CODE REVIEW

File: cmd/ddev/cmd/dotddev_assets/providers/pantheon.yaml.example [hunk 1]
```diff
@@ -64,18 +63,19 @@ files_pull_command:
...
+#    pushd /var/www/html/.ddev/.downloads >/dev/null;
+#    terminus remote:drush ${project} -- sql-drop -y
+#    gzip -dc db.sql.gz | terminus remote:drush ${project} -- sql-cli
+#
+## push is a dangerous command. If not absolutely needed it's better to delete these lines.
+#files_push_command:
+#  command: |
+#    # set -x   # You can enable bash debugging output by uncommenting
+#    ls ${DDEV_FILES_DIR} >/dev/null # This just refreshes stale NFS if possible
+#    terminus aliases 2>/dev/null
```
Comment (amitaibu) [inline cmd/ddev/cmd/dotddev_assets/providers/pantheon.yaml.example:80]: `terminus aliases` moved here

**Review by rfay — CHANGES_REQUESTED** [review]
* This approach will break the push tests, so we probably need to do something like adding an environment variable like "DDEV_ALLOW_PUSH=true" instead of the commenting-out technique you've done.
* We need to do it with all the example providers, not just this one.

**Comment by rfay** [comment]
This one is superceded by #2966 right @amitaibu ? Did you get everything you wanted in there and we should close this one?

**Comment by amitaibu** [comment]
Yes, this one can be closed. To be honest, I never got back to https://github.com/drud/ddev/pull/2966 , sorry :smiley_cat:

**Comment by rfay** [comment]
#2966 has excellent potential. One of us will get back to it :)

**Pull request closed without merging**